Src/VsVimShared/Implementation/Misc/StandardCommandTarget.cs

```diff
@@ -213,6 +215,17 @@ private bool TryProcessWithBuffer(KeyInput keyInput)
             return handled;
         }
 
+        private void CheckFocus()
+        {
+            // If, for some reason, keyboard input is being routed to the
+            // text view but it isn't focused, focus it here.
+            if (_vimBuffer.TextView is IWpfTextView wpfTextView && !wpfTextView.VisualElement.IsFocused)
+            {
+                VimTrace.TraceError("forcing focus back to text view");

```

Comment:
Why would this be an error? Is there something we are doing wrong which is changing the focus?

Reply:
It's not our error, but it is an error: the text view should not be receiving keystrokes unless it has the keyboard focus. But from a practical point of view, errors always get logged to the output window, so here `TraceError` means "trace important".

Reply:
Ha, ha, I don't know if nobody else notices these things besides me, they just ignore them, or if the problem is exclusive to my particular machine. I've noticed it for a long time, and it still occurs in both VS2017 and VS2019. I was suspicious that VsVim was the culprit, but after spending some time looking into it, I'm pretty sure it is a Visual Studio problem.

More background here: https://github.com/VsVim/VsVim/issues/2627#issue-454987003